runtests

```diff
@@ -2,8 +2,9 @@
 
 export NRNUNIT_USE_LEGACY=1
 
-sh testutil/clean
-neurondemo -c 'quit()'
+CURRENT_DIR=$(dirname $0)
```

Comment:
```suggestion
CURRENT_DIR=$(readlink $(dirname $0))
```
?

Reply:
not necessary to resolve path but why not...